guide/start/policies.md

```diff
@@ -9,169 +9,423 @@ children:
 - { section: "Managing with Policies", title_in_menu: "Policies"  } 
 ---
 
+## A Clustered Example
 
+We'll now look at a more complex example that better shows the capabilities of Brooklyn, including
+inlife management of a clustered application.
```

Comment:
I would hyphenate in-life or use different wording such as "including management of a running clustered application"